src/Scene/NodeProcess.js

```diff
@@ -253,48 +237,36 @@ function updateNodeImagery(scene, quadtree, node, layersConfig, force) {
             }
         }
 
-        let ancestor = null;
-
+        node.layerUpdateState[layer.id].newTry();
+        const searchInParent = !node.isColorLayerLoaded(layer.id) && node.parent.isColorLayerLoaded(layer.id);
         const currentLevel = node.materials[RendererConstant.FINAL].getColorLayerLevelById(layer.id);
-        // if this tile has no texture (level == -1), try use one from an ancestor
-        if (currentLevel === -1) {
-            ancestor = findAncestorWithValidTextureForLayer(node, l_COLOR, layer);
-        } else {
+
+        if (currentLevel > -1) {
             var targetLevel = chooseNextLevelToFetch(layer.updateStrategy.type, node.level, currentLevel, layer.updateStrategy.options);
-            if (targetLevel === currentLevel) {
+            if (targetLevel <= currentLevel) {
                 continue;
             }
-            if (targetLevel < node.level) {
-                ancestor = node.getNodeAtLevel(targetLevel);
-            }
         }
 
-        node.layerUpdateState[layer.id].newTry();
-
         const command = {
             /* mandatory */
             layer,
             requester: node,
             priority: nodeCommandQueuePriorityFunction(node),
             earlyDropFunction: refinementCommandCancellationFn,
-            /* specific params */
-            ancestor,
+            parentTextures: searchInParent ? node.parent.getLayerTextures(l_COLOR, layer.id) : undefined,
```

Comment:
> it should be seen in a new PR

This is something you add in *this* PR. I understand we delay refactoring of existing code, but here, this is not the case. Moreover, it seems to me that the goal of this PR was to remove ancestor, in order to remove the transitional state where a node has children, but they are not ready yet. If we achieve this original goal, this would greatly simplify the code and perform better. For this, this children preparation must be synchronous. 

Yes the scheduler prioritize things, but here, there is no notion of priority : it should be done now. Also, this scheduling has runtime cost (using an external lib, sorting the queue...). We should use it only when we must.